### PR TITLE
Now uses gradle

### PR DESCRIPTION
**Pull request by element54**
Converted everything to use gradle, added some documentation.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,21 @@
+apply plugin: 'java'
+apply plugin: 'eclipse'
+apply plugin: 'application'
+
+mainClassName = 'maas.Start'
+
+sourceCompatibility = 1.7
+targetCompatibility = 1.7
```
Comment (moriarty) [inline build.gradle]: Are we using java 7?

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,12 @@
+bin/
+.classpath
+.project
+.settings/
+build/
+.gradle/
+APDescription.txt
+MTPs-Main-Container.txt
```
Comment (moriarty) [inline .gitignore]: Are these two files generated by the build?

File: .travis.yml [hunk 3]
```diff
@@ -1,12 +1,6 @@
...
-script:
-    - export JADE_LIB=$TRAVIS_BUILD_DIR/lib
-    - export CLASSPATH=".:${JADE_LIB}/jade.jar:${JADE_LIB}/commons-codec/commons-codec-1.3.jar:${JADE_LIB}/jadeExamples.jar:${JADE_LIB}"
-    - export MAGIC_PATH=./bin
-    - export CLASSPATH=$CLASSPATH:${MAGIC_PATH}
-    - mkdir bin
-    - javac -d bin/ src/*/*.java
-    - java jade.Boot -agents Tester:maas.BookBuyerAgent
+group: stable
+dist: trusty
```
Comment (moriarty) [inline .travis.yml]: Ah- that's why we're using java7... :(

**Comment by moriarty** [comment]
👍 this looks good.

@element54 thanks.

@argenos you can merge this if you approve.

**Comment by argenos** [comment]
:+1: Also think so! Merged!

**Pull request merged**